clients/web/src/stylesheets/widgets/accessWidget.styl

```diff
@@ -18,16 +20,15 @@
       color #d0b822
 
 .g-ac-list-title
-  background-color #666
+  background-color #888
```

Comment:
Just an idle question: I've noticed a trend to making web pages lower contrast.  It doesn't matter much here, but is there some logic to it?